services/apps/test/apps/test_fixtures.clj

```diff
@@ -26,16 +15,7 @@
   (let [config-path (getenv "APPS_CONFIG_PATH" default-config-path)]
     (require 'apps.util.config :reload)
     (apps.util.config/load-config-from-file config-path {:log-config? false})
-    (f)))
-
-(defn with-service-uris [f]
```

Comment:
This fixture wasn't being used, and I couldn't think of a good reason to keep it around now that we have `with-config`.